repos/spack_repo/builtin/packages/kokkos/package.py

```diff
@@ -275,7 +277,8 @@ class Kokkos(CMakePackage, CudaPackage, ROCmPackage):
     variant("pic", default=False, description="Build position independent code")
 
     conflicts("cxxstd=11")
-    conflicts("cxxstd=14", when="@4.0:")
+    conflicts("cxxstd=14", when="@4:")
```

Comment:
The requirement actually goes back to Kokkos 3.3.00, back in the day we made less consistent decision with respect to backward compatibility I guess?
See here: https://github.com/kokkos/kokkos/blob/3.3.00/cmake/kokkos_pick_cxx_std.cmake
```suggestion
    conflicts("cxxstd=14")
```